### PR TITLE
pair work

### PR DESCRIPTION
**Pull request by markotuya0**
Use jest framework for testing.
-Create a test file ([..].test.js) for a file containing the add item and delete item functions that you want to test.
-Mock a storage object to "imitate" localStorage operations (only if you are using jest < 24.x, in newer versions localStorage is mocked automatically).
-Mock HTML to test if add/delete functions add or remove exactly one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,126 @@
...
-👤 **Author1**
+👤 **Yuriy Chamkoriyski**
 
-**Mehria-Rahmani***
-- Github: [@Merhia-Rahmani](https://github.com/Mehria-Rahmani)
-- Twitter: [@MehriaRahmani](https://twitter.com/MehriaRahmani?t=QwKwWXTVBZthv4sUrOXtfQ&s=08)
-- LinkedIn: [@mehria-rahmani](https://www.linkedin.com/in/mehria-rahmani-379211222) 
+- GitHub: [@Hombre2014](https://github.com/Hombre2014)
+- Twitter: [@Chamkoriyski](https://twitter.com/Chamkoriyski)
+- LinkedIn: [axebit](https://linkedin.com/in/axebit)
```
Comment (zhadier) [inline README.md]: - [ ] Kindly add the names of the current authors of this Project inside the ReadMe

File: license.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Copyright 2021, [Yuriy Chamkoriyski]
```
Comment (zhadier) [inline license.md]: - [ ] Kindly add the name of the current Author inside your MIT.md file so users know who has copyright claims for this project.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Mehria-Rahmani & @markotuya0  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️

- [ ] Please add a more detailed PR description for your Project which tells the Reviewer exactly what features you added in this branch.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mehria-Rahmani** [comment]
Use jest framework for testing.
-Create a test file ([..].test.js) for a file containing the add item and delete item functions that you want to test.
-Mock a storage object to "imitate" localStorage operations (only if you are using jest < 24.x, in newer versions localStorage is mocked automatically).
-Mock HTML to test if add/delete functions add or remove exactly one

**Comment by Mehria-Rahmani** [comment]
Use jest framework for testing.
-Create a test file ([..].test.js) for a file containing the add item and delete item functions that you want to test.
-Mock a storage object to "imitate" localStorage operations (only if you are using jest < 24.x, in newer versions localStorage is mocked automatically).
-Mock HTML to test if add/delete functions add or remove exactly one

**Pull request closed without merging**